src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -256,15 +256,35 @@ def _get_value(item, *args):
     except (KeyError, TypeError, IndexError):
         return ' '
 
-
-def _get_build_trigger(image_update_trigger, git_commit_trigger):
-    if git_commit_trigger.strip():
-        return 'Git Commit'
+def _get_statuses(item, *args):
+    """Get status of each source trigger.
+    :param dict item: The dict object
+    """
+    statuses = []
+    for arg in args:
+        item = item[arg]
```

Comment:
what does the code try to do in a loop `item = item[arg]`?

Reply:
This method takes in the json payload and iterated down to the nested 'sourceTriggers' type. From there it finds the status of every trigger in the task.